extensions/wikia/AdEngine/js/run/mercury.run.js

```diff
@@ -9,25 +9,29 @@ require([
 	'ext.wikia.adEngine.customAdsLoader',
 	'ext.wikia.adEngine.messageListener',
 	'ext.wikia.adEngine.mobile.mercuryListener',
+	'ext.wikia.adEngine.slotTweaker',
 	'ext.wikia.adEngine.slot.scrollHandler',
 	'ext.wikia.adEngine.provider.yavliTag',
 	'wikia.geo',
 	'wikia.instantGlobals',
 	'wikia.window'
-], function (adContext,
-			 amazon,
-			 oxBidder,
-			 prebid,
-			 rubiconFastlane,
-			 rubiconVulcan,
-			 customAdsLoader,
-			 messageListener,
-			 mercuryListener,
-			 scrollHandler,
-			 yavliTag,
-			 geo,
-			 instantGlobals,
-			 win) {
+], function (
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Function has 15 parameters which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)